### PR TITLE
Update kava-4 swagger

### PR DESCRIPTION
**Pull request by denalimarsh**
Update swagger for kava-4
- [x] auction module params 
- [x] add collateral-type to POST cdp txs
- [x] savings-rate-dist query
- [x] get cdps with query params

Based on `dm-queries` for review. **If https://github.com/Kava-Labs/kava/pull/644 is merged before this PR, the base branch must be changed to `master` before merging.**

### CODE REVIEW

File: swagger-ui/swagger.yaml [hunk 1]
```diff
@@ -441,6 +443,12 @@
             required: true
             type: string
             x-example: xrp
```
Comment (karzak) [inline swagger-ui/swagger.yaml:445]: Collateral denom has been replaced by collateral type -so we don't need both here.

File: swagger-ui/swagger.yaml [hunk 2]
```diff
@@ -533,6 +551,12 @@
...
```
Comment (karzak) [inline swagger-ui/swagger.yaml:553]: ditto

File: swagger-ui/swagger.yaml [hunk 3]
```diff
@@ -487,6 +497,12 @@
...
```
Comment (karzak) [inline swagger-ui/swagger.yaml:499]: ditto

File: swagger-ui/swagger.yaml [hunk 4]
```diff
@@ -577,6 +603,12 @@
...
```
Comment (karzak) [inline swagger-ui/swagger.yaml:605]: ditto

File: swagger-ui/swagger.yaml [hunk 5]
```diff
@@ -669,9 +702,18 @@
...
                   example: '1000000000'
+                surplus_auction_lot:
+                  type: string
+                  example: '10000000'
+                savings_distribution_frequency:
+                  type: string
+                  example: '60000000'
                 circuit_breaker:
                   type: boolean
                   example: false
```
Comment (karzak) [inline swagger-ui/swagger.yaml:719]: Can't highlight the right line here, but
```
/cdp/cdps/cdp/{owner}/{denom}
```

should become
```
/cdp/cdps/cdp/{owner}/{collateral-type}
```

Comment (karzak) [reply swagger-ui/swagger.yaml:719]: Lines 424, 478, 532, 584, 722, 749, 779, 815, 1397

File: swagger-ui/swagger.yaml [hunk 6]
```diff
@@ -805,6 +847,73 @@
...
+              properties:
+                height:
+                  type: string
+                  example: "100"
+                result:
+                  type: array
+                  items:
+                    $ref: '#/definitions/CdpResponse'
+          500:
+            description: Internal Server Error
```
Comment (karzak) [inline swagger-ui/swagger.yaml:916]: At line 1397, incentive should be updated to collateral-type. So `{denom}` becomes `{collateral-type}` and the example becomes `Collateral Type` with example `bnb-a`

**Review by karzak — CHANGES_REQUESTED** [review]

File: swagger-ui/swagger.yaml (hunk 1 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml:445]: Gotcha

Comment (denalimarsh) [reply swagger-ui/swagger.yaml]: https://github.com/Kava-Labs/kava/pull/653/commits/402223a6ab1c63ed0789c72e10d4d40ef2811a4f

File: swagger-ui/swagger.yaml (hunk 2 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml]: https://github.com/Kava-Labs/kava/pull/653/commits/402223a6ab1c63ed0789c72e10d4d40ef2811a4f

File: swagger-ui/swagger.yaml (hunk 3 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml]: https://github.com/Kava-Labs/kava/pull/653/commits/402223a6ab1c63ed0789c72e10d4d40ef2811a4f

File: swagger-ui/swagger.yaml (hunk 4 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml]: https://github.com/Kava-Labs/kava/pull/653/commits/402223a6ab1c63ed0789c72e10d4d40ef2811a4f

File: swagger-ui/swagger.yaml (hunk 6 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml:892]: https://github.com/Kava-Labs/kava/pull/653/commits/550065a767bc10dd230a8ee7f9a15a0b6742d1a3

File: swagger-ui/swagger.yaml (hunk 5 above)
Comment (denalimarsh) [reply swagger-ui/swagger.yaml:695]: https://github.com/Kava-Labs/kava/pull/653/commits/550065a767bc10dd230a8ee7f9a15a0b6742d1a3

**Review by karzak — APPROVED** [review]

**Pull request merged**